### PR TITLE
test prev cache key

### PR DESCRIPTION
**Pull request by deandreJones**
save correct cache key - update readme wording, testing for 508

### CODE REVIEW

**Review by traskowskycaci — COMMENTED** [review]
Do the restore_cache_for_go and save_cache_for_go also need to go from v7 to v8?

**Comment by deandreJones** [comment]
> Do the restore_cache_for_go and save_cache_for_go also need to go from v7 to v8?

only when we upgrade a go version

File: README.md [hunk 1]
```diff
@@ -4,9 +4,8 @@
 
 [![GoDoc](https://godoc.org/github.com/transcom/mymove?status.svg)](https://godoc.org/github.com/transcom/mymove)
 
-This repository contains the application source code for the Personal Property Prototype, a possible next generation version of the Defense Personal Property System (DPS). DPS is an online system managed by the U.S. [Department of Defense](https://www.defense.gov/) (DoD) [Transportation Command](http://www.ustranscom.mil/) (USTRANSCOM) and is used by service members and their families to manage household goods moves.
+This repository contains the application source code for the MilMove application,the next generation version of the Defense Personal Property System (DPS). DPS is an online system managed by the U.S. [Department of Defense](https://www.defense.gov/) (DoD) [Transportation Command](http://www.ustranscom.mil/) (USTRANSCOM) and is used by service members and their families to manage household goods moves.
```
Comment (danieljordan-caci) [inline README.md:7]: I hate to do this, but can you add a space after the comma here?
```
the MilMove application, the
```

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (deandreJones) [reply README.md:7]: why do you hate to do it.. I want it grammatically correct- I failed English, lol

Comment (danieljordan-caci) [reply README.md:7]: lol because it's a butterfly fart size of a change and this has been open since last week haha

Comment (deandreJones) [reply README.md:7]: every space matters

**Review by danieljordan-caci — APPROVED** [review]

**Pull request merged**